### ISSUE TITLE
Samsung Galaxy S20 FE - 

### ISSUE THREAD

**Issue by gittac** [koush/vysor.io#1274]
I can't find USB Debugging in Developer Settings. 
Is there some other setting as a work around?

**Comment by ajans** [comment]
No, but it's there, trust me (checked with my Samsung S20 FE aka SM-G780F/DS):
![grafik](https://github.com/koush/vysor.io/assets/3482744/9b4e7ff9-0422-4559-8348-ab33155532e2)

The setting can be found by searching it in system settings:
![grafik](https://github.com/koush/vysor.io/assets/3482744/5ef53fbf-cb74-4889-8697-34c55992d022)

Did you check to unlock the developer options?
